jobs/adspert_python_dev.html

```diff
@@ -0,0 +1,40 @@
+---
+title: Python Software Developer
+company: Adspert (Bidmanagement GmbH)
+url: https://www.adspert.net/en/jobs/python-software-developer-mf/
+location: Berlin, Germany
+contract: permanent
+contact:
+    name: Karl Bartel
+    email: jobs@adspert.net
+created: !!timestamp '2017-05-15'
+tags:
+  - python
+  - sql
+---
+We are looking for a **Python Software Developer (m/f)**
```

Comment:
For the BAML roles I have posted I typically include some boiler-plate text like:

"We are an equal opportunities employer and positively encourage applications from suitably qualified and eligible candidates regardless of sex, race, disability, age, sexual orientation, religion or belief."